src/store/getters.ts

```diff
@@ -282,4 +292,36 @@ export default {
     }
     return false;
   },
+  nftAssetsByCollection(...context: GetterContext): NFTAsset[] {
+    const { getters } = rootGetterContext(context);
+    const { nftAssetsByAccount } = getters;
+    let nftAssetsByCollection: NFTAsset[] = [];
+    // spread all nfts into one array
+    Object.values(nftAssetsByAccount).forEach((nftAssets) => {
+      nftAssetsByCollection = {
+        ...nftAssetsByCollection,
+        ...nftAssets,
+      };
+    });
+    return nftAssetsByCollection;
+  },
+  nftAssetsByAccount(...context: GetterContext): NFTAssets {
+    const { getters } = rootGetterContext(context);
+    const { accountsData } = getters;
+    const nftAssetsByAccount: NFTAssets = {};
+    accountsData.forEach((account) => {
+      if(account.nftAssets) {
+       const result = account.nftAssets.reduce(function (assets: NFTAssets, asset: NFTAsset) {
+            assets[asset.collection.name] = assets[asset.collection.name] || [];
+            assets[asset.collection.name].push(asset);
```

Comment:
the above 2 lines can be combiled with:

`(assets[asset.collection.name] ||= []).push(asset)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment